src/simularium/JsonFileReader.ts

```diff
@@ -0,0 +1,53 @@
+import type ISimulariumFile from "./ISimulariumFile";
+import type {
+    Plot,
+    SimulariumFileFormat,
+    TrajectoryFileInfo,
+    VisDataFrame,
+} from "./types";
+import { FrontEndError } from "./FrontEndError";
+import { compareTimes } from "../util";
+
+export default class JsonFileReader implements ISimulariumFile {
+    simulariumFile: SimulariumFileFormat;
+    constructor(fileContents: SimulariumFileFormat) {
+        this.simulariumFile = fileContents;
+
+        const spatialData = this.simulariumFile.spatialData;
+        if (!spatialData) {
+            throw new FrontEndError(
+                "Simularium files need 'spatialData' array"
+            );
+        }
+        spatialData.bundleData.sort(
+            (a: VisDataFrame, b: VisDataFrame): number =>
+                a.frameNumber - b.frameNumber
```

Comment:
did we do this sorting before? Is it not ok to assume they are in order?

Reply:
that code was there before, and I didn't have any reason to change it.  I don't know for sure but we probably have other implicit assumptions that they are in-order in the file.